### PR TITLE
Update from skeleton and update Fedora support

### PR DESCRIPTION
**Pull request by dav3r**
# 

## 🗣 Description
This PR pulls in the latest changes from [`skeleton-ansible-role`](https://github.com/cisagov/skeleton-ansible-role).

It also removes support for Fedora 29 and 31, and adds support for Fedora 32 and 33.

## 💭 Motivation and Context
I am preparing to update [`ansible-role-pca-gophish-composition`](https://github.com/cisagov/ansible-role-pca-gophish-composition) and noticed that this repo (which is used by `ansible-role-pca-gophish-composition`) was failing `molecule` testing, so I decided to do something about that.

## 🧪 Testing
All automated tests pass.

## 🚥 Types of Changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have updated tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: .github/lineage.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+version: "1"
+
+lineage:
+  skeleton:
+    remote-url: https://github.com/cisagov/skeleton-generic.git
```
Comment (mcdonnnj) [inline .github/lineage.yml]: Wrong skeleton I believe.

**Review by mcdonnnj — APPROVED** [review]
Looks good and makes sense to me 💪. Please see my comment about the Lineage configuration.

File: .github/lineage.yml (hunk 1 above)
Comment (dav3r) [reply .github/lineage.yml]: Gah, you are correct!  Thanks for the sharp 👀.  Fixed in https://github.com/cisagov/ansible-role-docker/pull/16/commits/f2adc1693c993c022ae4e7206524985c41b7f794.

Comment (felddy) [reply .github/lineage.yml]: I have nightmares about the wrong skeletons!  ☠️ 
Once a wrong skeleton bit me in the ass!  💀 🍑

File: molecule/default/tests/test_default.py [hunk 2]
```diff
@@ -20,6 +20,8 @@ def test_packages(host):
         # Debian Bullseye is not yet supported by the official Docker
         # package repo
         assert host.package("docker.io").is_installed
+    elif distribution == "fedora":
+        assert host.package("moby-engine").is_installed
```
Comment (felddy) [inline molecule/default/tests/test_default.py:24]: > And Moby, you can get stomped by Obie
You thirty six year old bald headed 🚬 , 💨  me
You don't know me, you're too old, let it go its over
Nobody listens to Techno

**Review by felddy — APPROVED** [review]
Looks good.  See my comments about Moby.  or don't.  🎵

**Pull request merged**